### DISCUSSION TITLE
Obliczanie kosztów pobranej energii

### DISCUSSION THREAD

**Discussion by Kutalon** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#161] [Q&A]
Mam w zakładce energia wybrany balanced consumption/ generation i wszystko działa elegancko, po zestawieniu z inwerterem oblicza autokonsumpcję itd.  Chciałem dodać obliczanie ceny pobranej i oddanej energii ale da się wybrać tylko: Nie śledź kosztów i Użyj encji do śledzenia całkowitych kosztów, Użyj encji z bieżącą ceną jest niedostępne. Coś robię źle czy wynika to z ograniczeń integracji?

**Comment by PiotrMachowski** [comment]
Najprawdopodobniej wynika to z faktu sposobu, w jaki pobierane są dane z Tauronu - jako statystyki, a nie jako sensory.
